src/Zoo.cpp

```diff
@@ -0,0 +1,486 @@
+/* This is a cpp file for the zoo class that runs the game.*/
+#include "Zoo.h"
+#include <vector>
+#include <cstdlib>
+#include <time.h>
+#include <iostream>
+#include <Dog.h>
+#include <Shark.h>
+#include <Clownfish.h>
+#include <sstream>
+#include <list>
+
+// Constructor
+Zoo::Zoo()
+{
+	for (int i = 0; i < 20; i++) {
+		for (int j = 0; j < 40; j++) {
+			map[i][j] = '-';
+		}
+	}
+
+	Animal::Location loc;
+	int rows = 20;
+	int columns = 40;
+
+	for (int i = 0; i < 3; i++)
```

Comment:
what is that ????
why using for loop and then checking the index value ??, what is benefit of using the  loop ??
what if you have 10 classes ??

Reply:
I fixed it.